### PR TITLE
feat: add possiblity to sort by updated_date and oldest collections

### PR DESCRIPTION
**Pull request by meelrossi**
Add new sorting possiblities to the collections list. Items added: `updated_at_asc`, `updated_at_desc` and `oldest` (this las one is to remain consistent with the `newest` option)

### CODE REVIEW

File: src/Collection/Collection.router.ts [hunk 1]
```diff
@@ -311,7 +311,7 @@ export class CollectionRouter extends Router {
         offset: page && limit ? getOffset(page, limit) : undefined,
         limit,
         address: eth_address,
-        sort: CurationStatusSort.NEWEST,
+        sort: sort as CurationStatusSort || CurationStatusSort.NEWEST,
```
Comment (cyaiox) [inline src/Collection/Collection.router.ts:314]: could you please add a test for this new query param? 🙏

File: src/Collection/Collection.router.spec.ts [hunk 2]
```diff
@@ -1059,12 +1060,15 @@ describe('Collection router', () => {
...
+        totalCollectionsFromDb: number,
+        sort: string
+  
       beforeEach(() => {
         ;(page = 1),
           (limit = 3),
           (isPublished = 'true'),
-          (totalCollectionsFromDb = 1)
+          (totalCollectionsFromDb = 1),
+          (sort=CollectionSort.NAME_ASC)
```
Comment (juanmahidalgo) [inline src/Collection/Collection.router.spec.ts:1071]:
```suggestion
          (sort = CollectionSort.NAME_ASC)
```

File: src/Collection/Collection.types.ts [hunk 3]
```diff
@@ -48,3 +48,13 @@ export enum CollectionTypeFilter {
   STANDARD = 'standard',
   THIRD_PARTY = 'third_party'
 }
+
+export enum CollectionSort {
+  MOST_RELEVANT = 'MOST_RELEVANT',
+  NEWEST = 'NEWEST',
```
Comment (juanmahidalgo) [inline src/Collection/Collection.types.ts:54]: I know this was already named like this, but now that we'll support both newest and oldest, just to keep it consistent with the rest of the enum options, what do you think if we name these ones as `CREATED_AT_DESC` and `CREATED_AT_ASC`? Then the front end will be the one translating newest and latest to `CREATED_AT_DESC` and ``CREATED_AT_ASC`

**Review by juanmahidalgo — COMMENTED** [review]
I just left two small comments but looks great 🚀

**Review by juanmahidalgo — APPROVED** [review]
looks great! feel free to merge whenever the other piece is ready 👍

**Pull request merged**